crates/bevy_pbr/src/volumetric_fog/mod.rs

```diff
@@ -104,7 +104,11 @@ impl Plugin for VolumetricFogPlugin {
                 Core3d,
                 // Volumetric fog is a postprocessing effect. Run it after the
                 // main pass but before bloom.
-                (Node3d::EndMainPass, NodePbr::VolumetricFog, Node3d::Bloom),
+                (
+                    Node3d::EndMainPass,
+                    NodePbr::VolumetricFog,
```

Comment:
I feel like maybe this should be ordered before EndMainPass if it's considered to be part of the main pass? I'm just not sure where it should go. I guess this works well enough though.

Reply:
I was 100% basing this based on the comment that was above 😅.